wp-content/themes/thestanforddaily/functions.php

```diff
@@ -149,6 +149,29 @@ function tsd_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'tsd_scripts' );
 
+/**
+ * Creates an excerpt if needed; and shortens the manual excerpt as well
+ */
+function wp_trim_all_excerpt($text) {
```

Comment:
Can you rename this to start with `tsd_` and also move this function to https://github.com/TheStanfordDaily/stanforddaily-website/blob/master/wp-content/themes/thestanforddaily/inc/template-functions.php ?